test/unit/expectation_test.rb

```diff
@@ -112,11 +112,9 @@ def test_should_yield_no_parameters
     assert_equal [], yielded_parameters
   end
 
-  def test_should_yield_with_specified_parameters
```

Comment:
I don't understand why you've removed this test where a block *is* passed to the stubbed method. Surely it's still relevant...?

Reply:
Oops! Restored.